doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -855,7 +855,10 @@ A subscription is represented by a JSON object with the following fields:
 + `id`: Subscription unique identifier. Automatically created at creation time.
 + `subject`: It is an object that describes the subject of the subscription.
 + `notification`: It is an object that describes the notification received by the subscriber.
-+ `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
++ `expires`: Subscription expiration date in ISO8601 format. Permanent subscriptions must omit
+   this field.
++ `status` (not editable, only in GET operations): Either `active` (for permanent subscriptions or if
```

Comment:
It would depend on how https://github.com/telefonicaid/fiware-orion/issues/1328 gets implemented at the end. Could you add a comment about this in that issue, pls?